### PR TITLE
Add triggers/scheduling to Lambda functions AUTO-803

### PR DESCRIPTION
**Pull request by aynsof**
Allow Lambda functions to be triggered on a Cloudwatch schedule - either cron-style or at a unit rate (e.g. '5 min').

Passing build: https://gaautobots.atlassian.net/builds/browse/AM-AUT79/latest

### CODE REVIEW

**Comment by GypsyBojangles** [comment]
What is description? :)

File: amazonia/application.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-amz_version: 1.3.6
+amz_version: 1.3.5
```
Comment (snayler0) [inline amazonia/application.yaml]: going back a version here. Was this just missed or is this intentional?

File: amazonia/application.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
 keypair: 'INSERT_YOUR_KEYPAIR_HERE'
 autoscaling_units:
```
Comment (GypsyBojangles) [inline amazonia/application.yaml]: this seems to have gone backwards

File: amazonia/classes/lambda_unit.py [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
-from troposphere import Ref
-from troposphere.awslambda import Code, VPCConfig, Function
+from troposphere import Ref, GetAtt
+from troposphere.awslambda import Code, VPCConfig, Function, Permission
+from troposphere.events import Rule, Target
 
 
 class LambdaUnit(SecurityEnabledObject):
-    def __init__(self, unit_title, template, dependencies, network_config, lambda_config):
+    def __init__(self, unit_title, template, dependencies, network_config, lambda_config, ):
```
Comment (GypsyBojangles) [inline amazonia/classes/lambda_unit.py]: extra comma

Comment (snayler0) [inline amazonia/classes/lambda_unit.py]: trailing comma

Comment (GypsyBojangles) [inline amazonia/classes/lambda_unit.py]: and we might need docstrings (sorry my bad) :)

**Comment by GypsyBojangles** [comment]
:+1:

**Comment by snayler0** [comment]
:shipit:

**Pull request merged**